addons/sourcemod/scripting/VIP_SourcemodFlags.sp

```diff
@@ -140,13 +140,13 @@ public Action OnClientPreAdminCheck(int client)
 
 public bool SBPP_OnClientPreAdminCheck(AdminCachePart part)
 {
-	if (part == AdminCache_Admins)

```

Comment:
i dont want to load vip clients if admins are not loaded

Reply:
Should we make a timer of 1s to let time for cache to be rebuild ? 
https://github.com/GFLClan/SM-User-Management/blob/b77579aa4a3c93fdbe12c9583bb61026e2f979d4/scripting/GFL-UserManagement.sp#L101

Reply:
we already have this forward SBPP_OnClientPreAdminCheck that is called when functions from OnRebuildAdminCache are done running